src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.Windows.cs

```diff
@@ -75,15 +75,13 @@ private void LoadSSPILibrary()
             }
         }
 
-        private void WaitForSSLHandShakeToComplete(ref uint error)
+        private void WaitForSSLHandShakeToComplete(ref uint error, ref int protocolVersion)
         {
-            if (TdsParserStateObjectFactory.UseManagedSNI)
-                return;
```

Comment:
Is there no possibility of control coming here for Managed SNI?

Reply:
This part is managed by overriding the `TdsParserStateObjectManaged.WaitForSSLHandShakeToComplete()`.